modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -293,6 +296,15 @@ public ObjectRelationship deleteObjectRelationship(
 				"Reverse object relationships cannot be deleted");
 		}
 
+		List<Long> aggregationFieldIds = _findAggregationFieldsIds(
+			objectRelationship.getObjectRelationshipId());
+
+		if (!aggregationFieldIds.isEmpty()) {
```

Comment:
Could you please check if we really need to add this condition?
I believe that the `.dslQuery(...)` method return an empty ArrayList if the query result is null. So we can just get the results and pass directly to the loop because the rows will never be null. Could you please check this?

Please let me know if you have any question